src/lib/server/websearch/generateQuery.ts

```diff
@@ -1,12 +1,25 @@
 import type { Message } from "$lib/types/Message";
+import { format } from "date-fns";
 import { generateFromDefaultEndpoint } from "../generateFromDefaultEndpoint";
 import { defaultModel } from "../models";
 
 export async function generateQuery(messages: Message[]) {
-	const promptSearchQuery = defaultModel.webSearchQueryPromptRender({ messages });
+	const currentDate = format(new Date(), "MMMM d, yyyy");
+	const lastMessage = messages[messages.length - 1];
```

Comment:
Because we use only the last message, you can't ask follow-up questions right?

Like I tried 
![image](https://github.com/huggingface/chat-ui/assets/25119303/fc7e4256-ce1e-4ccb-b4b1-847aa6345406) and it had no context of the previous question.

Not that big of a deal I think, since when we add the agents flow it will be solved, but just thought I'd bring it up.

Reply:
yes, mishig addressed this 🔥